docs/runtime_options.md

```diff
@@ -39,6 +39,16 @@ From [glog](https://github.com/golang/glog) here are some flags we find useful:
 --vmodule=: comma-separated list of pattern=N settings for file-filtered logging
 ```
 
+## Labels
+
+Both Docker and Rkt labels support restricting the amount of labels collected by `cadvisor`. 
```

Comment:
I would omit runtimes here as well: " `cadvisor` can restrict the set of labels it collects from containers.  This can be useful in ensuring metrics have a consistent set of labels."